script/Arcueid/Day 4/04_00_ARC04_5.txt

```diff
@@ -41,12 +44,15 @@ She also remembers going down of her own free will.
 [sha:0b221b9be711750015db9dc6623b4f8678bef421]{
 -- Page 345, Offset 6506.
 -- 昇降機を使った気もする。
-She also feels like she used an elevator.
+She's pretty sure she used an elevator.
 }
 [sha:be0ecb122451b4f449c01706807bb24c084d5168]{
 -- Page 345, Offset 6507.
 -- 落ちていく途中で、これが冗談のような都市伝説ではなく、本当に関わってはいけない都市の真実と気づいて、必死に停止ボタンを押し続けた気もする。
-On the way down, she realized that this was not some joke of an urban legend, but a truth about this city that she really shouldn't get involved in, and started pressing the stop button desperately.
+On the way down,
+ she realized that this was not some joke of an urban legend,
+ but a truth about this city that she really shouldn't get involved in,
+ and started frantically mashing the stop button.
```

Comment:
This last clause feels orphaned. Maybe this sentence should be restructured as:
> On the way down it became clear to her that this was not some joke or urban legend, but a truth about this city that she really shouldn't get involved in. When she realized that, she started frantically mashing the stop button.

or something to that effect